data-prepper-plugins/kafka-connect-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafkaconnect/util/SecretManagerHelper.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.kafkaconnect.util;
+
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
+import software.amazon.awssdk.core.client.config.ClientOverrideConfiguration;
+import software.amazon.awssdk.core.retry.RetryPolicy;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.secretsmanager.SecretsManagerClient;
+import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueRequest;
+import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueResponse;
+import software.amazon.awssdk.services.sts.StsClient;
+import software.amazon.awssdk.services.sts.auth.StsAssumeRoleCredentialsProvider;
+import software.amazon.awssdk.services.sts.model.AssumeRoleRequest;
+
+import java.util.UUID;
+
+public class SecretManagerHelper {
+    private static final String SESSION_PREFIX = "data-prepper-secretmanager-session";
+    public static String getSecretValue(final String stsRoleArn, final String region, final String secretId) {
```

Comment:
There is some other ongoing work to provide AWS Secrets Manager. See #2780. Let's use that instead.

Reply:
Ok, didn't know that. 
I can do a code clean up to use same SecretManager Utility to get secret after that secret-manager plugin is merged. but for this CR, let's keep it?

Reply:
I think it would be better to remove it in this PR so that we don't end up losing track of it. For testing purposes you could either:

* Use plain text temporarily
* Keep the changes you have locally.